actioncable/lib/action_cable/server/configuration.rb

```diff
@@ -29,7 +29,16 @@ def channel_class_names
           Pathname.new(channel_path).basename.to_s.split('.').first.camelize
         end
       end
+
+      # Returns constant of subscription adapter specified in config/cable.yml
+      # If the adapter cannot be found, this will default to the Redis adapter
+      def subscription_adapter
+        # Defaults to redis if no adapter is set
+        adapter = cable.fetch('adapter') { 'redis' }
+        adapter = adapter.camelize
+        adapter = "PostgreSQL" if adapter == "Postgresql"
```

Comment:
I think we went with "PostgreSQL" to have it match AR conventions. AR does the same thing, [here](https://github.com/rails/rails/blob/master/activerecord/lib/active_record/connection_adapters/connection_specification.rb#L37)